packages/viewer/src/lib/model-views/controller.ts

```diff
@@ -63,4 +67,68 @@ export class ModelViewController {
 
     return mapListItemModelViewsResponseOrThrow(res.toObject());
   }
+
+  /**
+   * Loads the provided model view within the current scene view.
+   *
+   * @param modelViewId The ID of the model view to load.
+   */
+  public async load(modelViewId: UUID.UUID): Promise<void> {
+    const scene = await this.sceneProvider();
+
+    await requestUnary(async (handler) => {
+      const deviceId = this.deviceIdProvider();
+      const meta = await createMetadata(this.jwtProvider, deviceId);
+      const req = new UpdateSceneViewRequest();
+
+      const svUuid = UUID.toMsbLsb(scene.sceneViewId);
+      const svUuid2l = new Uuid2l();
+      svUuid2l.setMsb(svUuid.msb);
+      svUuid2l.setLsb(svUuid.lsb);
+      req.setModelViewId(svUuid2l);
+
+      const mvUuid = UUID.toMsbLsb(modelViewId);
+      const mvUuid2l = new Uuid2l();
+      mvUuid2l.setMsb(mvUuid.msb);
+      mvUuid2l.setLsb(mvUuid.lsb);
+      req.setModelViewId(mvUuid2l);
+
+      const mask = new FieldMask();
+      mask.addPaths('sceneView.modelViewId');
```

Comment:
I'm not at all sure I understand the conventions around field masks, but looking at [this documentation](https://protobuf.dev/reference/protobuf/google.protobuf/#field-mask), it seems like the mask should be of the form `model_view_id`, so, not including the `sceneView` and using camel case for the field name (since this is gRPC Web rather than REST/JSON). Does that make sense to you (and @danschultz, if you have some opinions here)?

Reply:
Yeah, I think you're right. This at the very least seems like it needs to be snake cased rather than camel cased. https://github.com/Vertexvis/language-guides/blob/main/grpc/standard-fields.md suggests that it should be relative to the resource rather than the request, so I'm not certain if the update request implies we're operating on a `scene_view`, or if we should explicitly call that out

Reply:
BTW, the guide is incorrect as we decided to not use a "source" object.

Reply:
👍 good to know. Thanks!